test/integration/nginx_less_mpi_connection_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // Verify that the agent sends a connection request to Management Plane even when Nginx is not present
 func TestNginxLessGrpc_Connection(t *testing.T) {
-	teardownTest := setupConnectionTest(t, true, true)
+	teardownTest := setupConnectionTest(t, true, true, "../config/agent/nginx-config-with-max-file-size.conf")
```

Comment:
```suggestion
	teardownTest := setupConnectionTest(t, true, true, "../../config/agent/nginx-config-with-max-file-size.conf")
```
This path will need to change before `cleanup-integration-tests` pr is merged